tests/framework/installer/ceph_manifests.go

```diff
@@ -112,7 +113,12 @@ func (m *CephManifestsMaster) GetToolbox() string {
 		manifest = strings.ReplaceAll(manifest, "name: rook-direct-mount", "name: rook-ceph-tools")
 		return strings.ReplaceAll(manifest, "app: rook-direct-mount", "app: rook-ceph-tools")
 	}
-	return m.settings.readManifest("toolbox.yaml")
+	manifest := m.settings.readManifest("toolbox.yaml")
+	// The toolbox uses the ceph image, so replace the version that is being tested
+	// The regex allows for the versions such as "v18" or "v18.2.10" to be replaced
+	versionRegex := regexp.MustCompile(`image: quay.io/ceph/ceph:v[0-9]+[.0-9]+`)
```

Comment:
We do occasionally test main or another dev branch. Should we try this instead?

```suggestion
	versionRegex := regexp.MustCompile(`image: quay.io/ceph/ceph:.+`)
```

Reply:
We just need to match what is committed to the repo. Currently it's v17.2.6, and should remain a specific version. The replacement value will certainly be for a main or other tag. That said, it would be simpler just to allow any character for the tag as you suggested, thanks.